qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/completions/QuteSnippetCompletionInIfSectionTest.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.services.completions;
+
+import static com.redhat.qute.QuteAssert.SECTION_SNIPPET_SIZE;
+import static com.redhat.qute.QuteAssert.c;
+import static com.redhat.qute.QuteAssert.r;
+import static com.redhat.qute.QuteAssert.testCompletionFor;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for Qute snippet completion in #if section.
+ *
+ */
+public class QuteSnippetCompletionInIfSectionTest {
+
+	@Test
+	public void singleIf() throws Exception {
+		String template = "|";
+		testCompletionFor(template, //
+				true, // snippet support
+				c("if", //
+						"{#if ${1:condition}}\n" + //

```

Comment:
This test fails on Windows OS.

As original template content is `""` it doesn't define some line separator, in this case, the line separator comes from OS. So you need to use System.lineSeparator(). Please replace all \n with System.lineSeparator()

```
"{#if ${1:condition}}" + System.lineSperataor() + //
...

```